docs/semantics.md

```diff
@@ -0,0 +1,87 @@
+# RDF-star Semantics Options
+
+There have been several different semantics proposals for quoted triples, some with multiple semantics.  The proposed semantics differ in technical details but also differ in the entailments they support.  The major differences between semantics can be summarized in whether they are sensitive to the precise syntax of quoted triples or only to the semantics of the components of quoted triples.   The first kind of semantics have been called opaque and the second kind transparent.  There is also a semi-opaque (or semi-transparent) semantics, as in the community report, where the syntax of IRIs and literals are important but only the meaning of blank nodes is.
+
+The difference between the semantics can be seen in the following examples.
+
```

Comment:
```suggestion
Note that, although we use the Turtle syntax for convenience, we interpret it in a subtly different way: each blank node labels is interpreted  as specific and fixed element of the set of Blank Nodes defined by the RDF abstract syntax.

```
Without this comment, I would argue that it is impossible to decide whether the fisrt entailment holds or not in the opaque semantics (because we can't tell for sure if `_:a` and `_:f` are representing the same or different blank nodes...

Reply:
Good to point it out. As defined in syntax currently, the scope of labels is the document. Same label, same blank node entity. Interpretation of the blank node may differ which is not a syntax issue.

Reply:
Agreed.  I've added this to my local copy and will upload shortly.